tmva/tmva/inc/TMVA/Executor.h

```diff
@@ -46,9 +46,13 @@ class Executor {
    //////////////////////////////////////
    /// Default constructor of TMVA Executor class
    /// if ROOT::EnableImplicitMT has not been called then by default a serial executor will be created
-   /// A user can create a thread pool and enable multi-thread execution by calling TMVA::Config::Instance()::EnableMT(nthreads)
-   /// For releasing the thread pool used by TMVA one can do it by calling  TMVA::Config::Instance()::DisableMT() or
-   /// calling TMVA::Config::Instance()::EnableMT() with only one thread
+   /// A user can create a thread pool and enable multi-thread excution by calling
+   /// 
+   ///     TMVA::Config::Instance()::%EnableMT(int nthreads);
```

Comment:
```suggestion
   /// ~~~{.cpp}
   /// TMVA::Config::Instance()::%EnableMT(int nthreads);
   /// ~~~
```
Less confusing then indent, and you can do syntax highlighting (yes, it's apostrophes in markdown and tildes in doxygen :()